src/main/java/Game.java

```diff
@@ -1,14 +1,18 @@
 import enums.TileType;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import java.util.Scanner;
 
 public class Game {
     private int turns;
     private Player[] players;
+    private String[] mapHTML;
```

Comment:
Why is this a global variable? Didn't we agree that generateHtml will call player.getMap().generateHtml every time?